hack/rp-dev/rp_funcs.sh

```diff
@@ -0,0 +1,197 @@
+#!/bin/bash -e
+######## Helper file with the functions to fully deploy RP dev either locally or using Azure DevOps Pipelines ########
+
+source hack/devtools/rp_dev_helper.sh
+
+setup_rp_config() {
+  # Check if exactly non-empty two arguments are provided
+  if [[ $# -ne 2 ]]; then
+    echo "Error: $0 <AZURE_PREFIX> <SKIP_DEPLOYMENTS>. Please try again"
+    exit 1
+  fi
+
+  # TODO - check if needed
+  export USER=dummy
+  # Export the SECRET_SA_ACCOUNT_NAME environment variable and run make secrets
+  az account set -s fe16a035-e540-4ab7-80d9-373fa9a3d6ae
+  export SECRET_SA_ACCOUNT_NAME=rharosecretsdev && make secrets
```

Comment:
Exporting isn't necessary here.

```suggestion
  SECRET_SA_ACCOUNT_NAME=rharosecretsdev make secrets
```